src/components/Tooltip/Tooltip.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="relative">
+    <div
+      :class="[
```

Comment:
Nitpick: can we (1) condense this `class` attribute to one line since it's short and (2) remove the array since it doesn't look like we need it to be an array?